packages/template-ui/src/components/Breadcrumb.vue

```diff
@@ -1,28 +1,43 @@
 <template>
   <b-navbar-nav
-    v-if="(node.ancestors && node.ancestors.length) || notAtHome"
     class="mt-3"
     aria-label="breadcrumb"
   >
-    <ChannelLogo class="mt-1" :channel="channel" size="sm" />
+    <ChannelLogo v-if="!xs" class="mt-1" :channel="channel" size="sm" />
     <ol
       v-if="node.ancestors && node.ancestors.length"
       class="bg-transparent breadcrumb px-2"
     >
-      <li
-        v-for="a in node.ancestors"
-        :key="a.id"
-        class="breadcrumb-item"
-      >
-        <b-link
-          :to="getTopicUrl(a)"
+      <template v-if="xs || sm">
+        <li class="active breadcrumb-item">
```

Comment:
This "three dots" element is shown always, even if there's only one item in the breadcrumb, in that case we shouldn't show it.

```suggestion
        <li v-if="node.ancestors.length > 1" class="active breadcrumb-item">
```